dot/node.go

```diff
@@ -317,7 +317,18 @@ func NewNode(cfg *Config, ks *keystore.GlobalKeystore) (*Node, error) {
 	// check if rpc service is enabled
 	if enabled := cfg.RPC.isRPCEnabled() || cfg.RPC.isWSEnabled(); enabled {
 		var rpcSrvc *rpc.HTTPServer
-		rpcSrvc, err = createRPCService(cfg, ns, stateSrvc, coreSrvc, networkSrvc, bp, sysSrvc, fg)
+		cRPCParams := createRPCServiceParams{
+			cfg,
+			ns,
+			stateSrvc,
+			coreSrvc,
+			networkSrvc,
+			bp,
+			sysSrvc,
+			fg,
+			syncer,
+		}
```

Comment:
Use named fields please.
Also please rename `createRPCServiceParams` to `rpcServiceSettings` for example

Reply:
Done in 3de34da